Code/Editor/EditorEngineProcess/EngineProcGameApp.cpp

```diff
@@ -301,7 +311,10 @@ void ezEngineProcessGameApplication::EventHandlerIPC(const ezEngineProcessCommun
     }
     else if (pMsg1->m_sWhatToDo == "ReloadResources")
     {
-      ezResourceManager::ReloadAllResources(false);
+      if (pMsg1->m_sPayload == "ReloadAllResources")

```

Comment:
Why is the DeleteAllCachedRenderData only in the one case? Are there other cases, or should there be an assert_not_implemented in the else case ?

Reply:
So if you click "reload resources" in the menu, it will set the payload to "ReloadAllResources" and behave as before, i.e. calling:
```
ezResourceManager::ReloadAllResources(false);
ezRenderWorld::DeleteAllCachedRenderData();
```
Now if I incrementally reload resources in the background I still need to invalidate the cached render data at some point. I didn't want to do that on every resource so instead I fire the "ReloadResources" message but without the payload. Not sure if that is the ideal process of dealing with this problem.